global.json

```diff
@@ -3,7 +3,7 @@
     "dotnet": "3.0.100-preview7-012821"
   },
   "native-tools": {
-    "cmake": "3.11.1",
+    "cmake": "3.14.2",
```

Comment:
Is this actually doing anything as native tools installation is disabled: https://github.com/dotnet/coreclr/blob/4e702da2c06ded8239d5eb2f47619604efbf3fde/eng/configure-toolset.ps1#L12?

Reply:
We manually call the init-native-tools.ps1 script in build.cmd and build-test.cmd

Reply:
I see, thanks.